src/main/scala/terminal/Files.scala

```diff
@@ -0,0 +1,5 @@
+package terminal
+
+class Files(name : String) extends Folders(name) {
```

Comment:
Usually classes have singular names e.g: `File`, `Folder` since an object of this type represents only a single file, not multiple ones. Also, inheritance models a IS A relationship. A File is not a Folder since you cannot have a file that contains files inside it.